source/mir/blas/README.md

```diff
@@ -0,0 +1 @@
+BLAS is not ready. It is slow. It is not tested. Cache oriented BLAS would be anonced at https://forum.dlang.org .
```

Comment:
"BLAS is not ready" -> "BLAS is still in development and will undergo more work & testing. Use at your own risk. Cache oriented BLAS would be anonced at https://forum.dlang.org and [Twitter](https://twitter.com/libmir).